src/wallet/rpcwallet.cpp

```diff
@@ -124,9 +124,12 @@ void EnsureWalletIsUnlocked(const CWallet* pwallet)
     }
 }
 
-LegacyScriptPubKeyMan& EnsureLegacyScriptPubKeyMan(CWallet& wallet)
+LegacyScriptPubKeyMan& EnsureLegacyScriptPubKeyMan(CWallet& wallet, bool also_create)
```

Comment:
What's the general usage for this new boolean? I see it's set to true for most importing, but not for `addmultisigaddress` for instance. A guiding comment on usage would be helpful.

Reply:
Added a comment.

It's to be used for RPCs that make a blank wallet no longer blank. `addmultisigaddress` wasn't doing this before, although I suppose the expected behavior of it with a blank wallet isn't really known (i.e. what do we want it to do?).

Reply:
Hm, yeah it would be a weird non-blank wallet. I'm fine as-is for now.